### PR TITLE
Added auto commenting featue on new issue and pr

### LINKED ISSUE #1075: Feature for auto commenting on a new issue creation
I would like an automated feature that posts a predefined comment on an issue as soon as it is created. This feature aims to address the problem of delayed and inconsistent communication following the creation of issues. By automatically commenting on issues as soon as they are created, it ensures that contributors receive immediate feedback and acknowledgment for their efforts. This fosters a positive and supportive environment for collaboration, encouraging continued participation and engagement from contributors.

Could you please assign me this issue under GSSOC'24.

### PR DESCRIPTION
**Pull request by suhanipaliwal**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

I have added auto-comment-pr-raise.yml in the .github/workflows directory which will add the feature to auto comment on pull request being raised.

Fixes #1075 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: .github/workflows/auto-comment-pr-raise.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  comment:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Add Comment to Pull Request
+      run: |
+        COMMENT=$(cat <<EOF
+        {
+          "body": "Thank you for submitting your pull request! We'll review it as soon as possible. For further communication, join our discord server https://discord.gg/tSqtvHUJzE."
+        }
```
Comment (Kushal997-das) [inline .github/workflows/auto-comment-pr-raise.yml:21]: It seems you did copy-paste from somewhere, we sadly do not have any discord but you did added.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@suhanipaliwal It's seems you have copy the code from somewhere, kindly do the stuff according to my repo.

**Comment by suhanipaliwal** [comment]
I'm really sorry for this mistake. I've done the necessary changes now.

**Review by Kushal997-das — APPROVED** [review]
@suhanipaliwal If it's work then only i'll put the level after merge.

**Comment by suhanipaliwal** [comment]
Please add the label of level1 on this pull request.

**Comment by suhanipaliwal** [comment]
Could you please add the label of level1

**Pull request merged**